antibody.go

```diff
@@ -28,29 +39,97 @@ func New(home string, r io.Reader) *Antibody {
 
 // Bundle processes all given lines and returns the shell content to execute
 func (a *Antibody) Bundle() (result string, err error) {
-	var g errgroup.Group
-	var lock sync.Mutex
-	var shs []string
-	scanner := bufio.NewScanner(a.r)
-	for scanner.Scan() {
-		l := scanner.Text()
-		g.Go(func() error {
-			l = strings.TrimSpace(l)
-			if l != "" && l[0] != '#' {
-				s, err := bundle.New(a.Home, l).Get()
-				lock.Lock()
-				shs = append(shs, s)
-				lock.Unlock()
-				return err
+	file := a.r
```

Comment:
Yeah, first versions of antibody were using channels too, you can check the history if you're curious :P